wx/lib/stattext.py

```diff
@@ -303,7 +303,7 @@ def OnPaint(self, event):
                 x = width - w
             if style & wx.ALIGN_CENTER:
                 x = (width - w)//2
-            dc.DrawText(line, x, y)
+            dc.DrawText(line, int(round(x)), int(round(y)))
```

Comment:
To preserve existing behavior, I think it should just be: `dc.DrawText(line, int(x), int(y))`

Reply:
No problem - PR updated!